tests/test_cli.py

```diff
@@ -239,7 +239,37 @@ def test_image_files_overwritten_if_no_output_dir(integration):
     assert shape == (500, 500, 3)
 
 
-def test_main_ImageReadError():
-    """Monketpatch an ImageReadError to main to trigger those lines"""
-    # TODO: that line of inquiry should also reject_count += 1
-    pass
+@pytest.mark.parametrize(
+    "extension, error_expected, expected",
+    [
+        (".png", False, "png"),
+        ("png", False, "png"),
+        ("PNG", False, "png"),
+        ("fake_ext", True, None),
```

Comment:
Here I set up all the test cases I feel are important to your function. I set them as parameters to the test function definition on line 253. This makes us avoid repeating the test code.